Created .env environment configuration file

hwarnuh opened the pull request:
This .env file specifies the necessary Flask environment variables to run the scraping server with in conjunction with Michael's updates in the flask-cors-fix branch. A user must edit line 3 with the confidential Google OAuth Client ID before running the server with Flask (which is reflected in the edited README.md on this branch).

hwarnuh committed:
Created .env environment configuration file

This .env file specifies the necessary Flask environment variables to run the scraping server with in conjunction with Michael's updates in the flask-cors-fix branch. A user must edit line 3 with the confidential Google OAuth Client ID before running the server with Flask (which is reflected in the edited README.md on this branch).

hwarnuh committed:
Update README with .env instructions

In the add-flask-env branch, I have created a .env file containing the necessary Flask environment variables to run the scraping server. In this commit, I updated the README with instructions to initialize the environment, as well as editing the Flask and React terminal commands to run the server on macOS.

hwarnuh committed:
Update README with clarifications

On the add-flask-env branch, I have added a .env file that creates the necessary Flask variables. Thus, in the README, I added clarifications to the prior environment variable initialization.

jhaber-zz commented:
Looks great! Holding off on approval until install setup changes to Ubuntu have been implemented in README.

hwarnuh committed:
Updated Ubuntu Instructions

Added new instructions for running the scraping server with Ubuntu - edited Flask and React server commands and added information about .env file.
